ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml

```diff
@@ -238,71 +238,77 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: rosa-aws-sts-advanced
-- as: aws-rosa-sts-private-link-f7
-  cron: 22 20 6,14,21,28 * *
+- as: aws-rosa-sts-hcp-sector-prod-critical-f7
+  cron: 28 6 3,10,17,26 * *
   steps:
-    cluster_profile: aws-qe
+    cluster_profile: aws-sd-qe
     env:
       CHANNEL_GROUP: candidate
+      CLUSTER_SECTOR: canary
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @rosa and @hypershift-hosted and @critical
+        and not @fips'
```

Comment:
`not @fips` is no longer needed, more detail in https://github.com/openshift/release/pull/47129 and https://github.com/openshift/release/pull/47158

Reply:
Thanks for your remind, could you help to review it again? Thanks